sync/projects.py

```diff
@@ -0,0 +1,51 @@
+from sync.api.projects import get_submissions
+
+
+class SubmissionRetrievalError(Exception):
+    """Custom exception for errors retrieving submissions."""
+
+
+class NoSubmissionsFoundError(SubmissionRetrievalError):
+    """No submissions were found."""
+
+
+class NoSuccessfulSubmissionsFoundError(SubmissionRetrievalError):
+    """No successful submissions were found."""
+
+
+def get_latest_submission_config(project_id: str, success_only: bool = False) -> dict:
```

Comment:
I think we want success by default

Reply:
i wasnt sure on that to be honest. That seems like a preference thing right? Like latest inherently means the latest without conditions, no? Happy to change it though